LiveStream/ViewModels/LiveVideoViewModel.swift

```diff
@@ -42,6 +48,15 @@ internal protocol LiveVideoViewModelOutputs {
 
   /// Emits a stream when the subscriber of that stream should be removed.
   var removeSubscriber: Signal<OTStreamType, NoError> { get }
+
+  /// Emits when all subscribers should be re-subscribed when the view reappears.
+  var resubscribeAllSubscribersToSession: Signal<(), NoError> { get }
+
+  /// Emits to toggle play/pause when the view disappears/reappears.
+  var setHlsToPlaybackState: Signal<Bool, NoError> { get }
```

Comment:
oh lol didn't realize @mbrandonw commented on this already. I agree we shouldn't use "toggle" but we should use "pause" because otherwise it's not clear what state this refers to.